src/System.Windows.Forms.Primitives/tests/Interop/Oleaut32/IDispatchTests.cs

```diff
@@ -0,0 +1,97 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing;
+using Xunit;
+using static Interop;
+using static Interop.Ole32;
+using static Interop.Oleaut32;
+
+namespace System.Windows.Forms.Primitives.Tests.Interop.Oleaut32
+{
+    [Collection("Sequential")]
+    public class IDispatchTests : IClassFixture<ThreadExceptionFixture>
```

Comment:
Should this be https://xunit.net/docs/shared-context#collection-fixture?
```suggestion
    public class IDispatchTests : ICollectionFixture<ThreadExceptionFixture>
```

Reply:
> A test class may not be decorated with ICollectionFixture<> (decorate the test collection class instead).

FYI